weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/traceServerCachingClient.ts

```diff
@@ -0,0 +1,106 @@
+import LRUCache from 'lru-cache';
+
+import {
+  TraceFileContentReadReq,
+  TraceFileContentReadRes,
+  TraceObjReadReq,
+  TraceObjReadRes,
+  TraceTableQueryReq,
+  TraceTableQueryRes,
+  TraceTableQueryStatsReq,
+  TraceTableQueryStatsRes,
+} from './traceServerClientTypes';
+import {DirectTraceServerClient} from './traceServerDirectClient';
+
+interface CacheConfig {
+  maxSize: number;
```

Comment:
It looks like lru-cache can be configured with either `max` or `maxSize`, we declare it as `maxSize` but then use it as `max` which is kind of confusing.

Reply:
nice - fixed